File: SecondYear/semestrul4/MPP/examen_practic/Rezolvari/SUB3_2024/services/src/main/java/org/services/ProjectException.java
package org.services;
import java.io.Serializable;
public class ProjectException extends Exception implements Serializable {
    public ProjectException() {
    }

    public ProjectException(String message) {
        super(message);
    }

    public ProjectException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: SecondYear/semestrul4/MPP/PROIECT/CSharpProjectNetworking/Services/ProjectException.cs

using System.Runtime.Serialization;

namespace Services;

public class ProjectException : Exception
{
    public ProjectException() : base()
    {
    }

    public ProjectException(string message) : base(message)
    {
    }

    public ProjectException(string message, Exception innerException) : base(message, innerException)
    {
    }
    
    public ProjectException(SerializationInfo info, StreamingContext context) : base(info, context)
    {
    }
}
